packages/vanilla/src/components/dotcomshell/README.md

```diff
@@ -0,0 +1,118 @@
+# DotcomShell
+
+> The DotcomShell component includes the `Masthead`, and `Footer` components,
+> all wrapped in a UI shell using Carbon's grid.
+
+## Getting started
+
+```scss
+// yourapplication.scss
+@import '@carbon/type/scss/font-face/mono';
+@import '@carbon/type/scss/font-face/sans';
+@include carbon--font-face-mono();
+@include carbon--font-face-sans();
+```
+
+> 💡 Only import font's once per usage
+
+### Content
+
+```javascript
+import 'yourapplication.scss';
+
+const content = `
+  <p>Your content here (probably include something nicer than this! 😄)</p>
+`;
+
+export default content;
+```
+
+### DotcomShell
+
+```javascript
+import { DotcomShell } from '@carbon/ibmdotcom-vanilla';
+import '@carbon/ibmdotcom-styles/scss/components/dotcom-shell/_dotcom-shell.scss';
```

Comment:
@kennylam this should be under `yourapplication.scss` right?

Reply:
@jeffchew ah yes, good catch